reader-gtfs/src/test/java/com/graphhopper/ExtendedRouteTypeIT.java

```diff
@@ -48,6 +48,8 @@ public static void init() {
         ghConfig.put("graph.flag_encoders", "car,foot");
         ghConfig.put("graph.location", GRAPH_LOC);
         ghConfig.put("gtfs.file", "files/another-sample-feed-extended-route-type.zip");
+        // todonow: do we want the (previously by default) CH profiles here? (same question for all GraphHopperGtfs basically
+        ghConfig.put("prepare.ch.weightings", "fastest");
```

Comment:
@michaz ? I wasn't sure if CH was used for this test only by accident (because of the default) or if its actually intended.

Reply:
Seems like an accident, pt does not use CH -> removed this